code/common/os.q

```diff
@@ -14,4 +14,5 @@ df:{(`$("/";"\\")[NT]sv -1_v;`$-1#v:("/";"\\")[NT]vs pth(string x;x)[10h=type x]
 run:{system"q ",x}
 kill:{[p]@[(`::p);"\\\\";1];}
 sleep:{x:string x; system("sleep ",x;"timeout /t ",x," >nul")[NT]}
-pthq:{[x] $[10h=type x;ssr [x;"\\";"/"];`$ -1 _ ssr [string (` sv x,`);"\\";"/"]]}
\ No newline at end of file
+pthq:{[x] $[10h=type x;ssr [x;"\\";"/"];`$ -1 _ ssr [string (` sv x,`);"\\";"/"]]}
+list:{$[NT;system"dir ",pth x;system"ls ",pth x]}

```

Comment:
Again duplication can be removed:
```
{system$[NT;"dir ";"ls "],pth x}
```

Reply:
I think this whole function can probably be removed, as @jonnypress pointed out `key` can & should be used instead